config/cageConfiguration.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" ?>
+<cageConfiguration>
+	<cageStructure>
+		<height>200</height>
+		<radius>60</radius>
+	</cageStructure>
+	<muscleUnits>
+		<muscleUnit name="motor0">
+			<height>102</height>
+			<angle>20</angle>
+		</muscleUnit>
+		<muscleUnit name="motor1">
+			<height>102</height>
+			<angle>80</angle>
+		</muscleUnit>
+		<muscleUnit name="motor2">
+			<height>60</height>
+			<angle>170</angle>
+		</muscleUnit>
+	</muscleUnits>
+	<tendons>
+		<tendon name="tendon0">
```

Comment:
each tendon has at least 2 points - start and end

Reply:
in what frame are these coordinates defined?

Reply:
the coordinates where defined in the link frame

Reply:
it will be needed, the way it is now in exoforce means the link's center of mass, 
wrist, for example, will have many tendons attached to it and it will not be a center of mass and they will not be identical positions

Reply:
in exoforce the motor position (or the other end of the tendon) defined in the world frame though, you have to think of a good way of defining it

Reply:
shouldn't it be defined in the cage frame because it will rotate?

Reply:
think about it and decide what makes the most sense. that's why a raised a question about frames... exoforce is a quick prototype to get things running, right...

Reply:
requires further discussion